src/energyplus/ForwardTranslator/ForwardTranslateZoneHVACTerminalUnitVariableRefrigerantFlow.cpp

```diff
@@ -278,6 +281,8 @@ namespace energyplus {
         return std::pair<unsigned, unsigned>{Fan_ConstantVolumeFields::AirInletNodeName, Fan_ConstantVolumeFields::AirOutletNodeName};
       } else if (idf_fan.iddObject().type() == IddObjectType::Fan_SystemModel) {
         return std::pair<unsigned, unsigned>{Fan_SystemModelFields::AirInletNodeName, Fan_SystemModelFields::AirOutletNodeName};
+      } else if (idf_fan.iddObject().type() == IddObjectType::Fan_VariableVolume) {
+        return std::pair<unsigned, unsigned>{Fan_VariableVolumeFields::AirInletNodeName, Fan_VariableVolumeFields::AirOutletNodeName};
```

Comment:
surprised to see this addition here. Could you explain why this was needed please?

Reply:
From `ZoneHVAC:TerminalUnit:VariableRefrigerantFlow`:

```
  A7 ,  \field Supply Air Fan Object Type
        \type choice
        \key Fan:SystemModel
        \key Fan:OnOff
        \key Fan:ConstantVolume
        \key Fan:VariableVolume
        \default Fan:ConstantVolume
        \note Supply Air Fan Object Type must be Fan:SystemModel, Fan:OnOff, or Fan:ConstantVolume
        \note if AirConditioner:VariableRefrigerantFlow is used to model VRF outdoor unit
        \note Supply Air Fan Object Type must be Fan:SystemModel or Fan:VariableVolume if
        \note AirConditioner:VariableRefrigerantFlow:FluidTemperatureControl or
        \note AirConditioner:VariableRefrigerantFlow:FluidTemperatureControl:HR
        \note is used to model VRF outdoor unit
        \note Required for zone equipment. Leave blank if terminal unit is used in AirLoopHVAC:OutdoorAirSystem:EquipmentList.
        \note Also leave blank if terminal unit is used on main AirloopHVAC branch and terminal unit has no fan.
```

Reply:
but was it not working before then? If so we should make a unit test.

Reply:
It was not working before this addition. For example, if I comment out this code, the FT test for `AirConditionerVariableRefrigerantFlowFluidTemperatureControl` fails (on account of having a terminal with a variable volume fan).

```
[openstudio.energyplus.ForwardTranslator] <1> VRF named Zone HVAC Terminal Unit Variable Refrigerant Flow 1 uses an unsupported fan type.
```